wordpress-mobile/WordPress-iOS
# Jetpack Section: Scan: Adds threat fixing status

Project: #15190 
---

Related: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/342

Adds a new state that displays the threats that are actively being fixed. 

### Demo
https://user-images.githubusercontent.com/793774/106516007-7842f400-64a4-11eb-8f8a-46d6bf8caeb8.mov

### To test:

**Note:** You can generate a fixable threat by installing these plugins: 
- [calendar-1.zip](https://github.com/wordpress-mobile/WordPress-iOS/files/5906694/calendar-1.zip)
- [wordpress-seo.zip](https://github.com/wordpress-mobile/WordPress-iOS/files/5906708/wordpress-seo.zip)

1. Launch the app, tap on the My Site, tap on a site with Jetpack Scan
2. Tap Scan
3. Tap the 'Fix All' button
4. 👁️ You should immediately see the fixing status with all the fixable threats displayed
5. The threat description should inform you as to how Jetpack will auto fix it
5. The view will auto refresh after 5 seconds, and should appear to be the same
6. Pull to refresh to verify you're still in the fixing status
7. Wait a few seconds for the threat to be fixed
8. If it's not the last threat you should see it removed from the list
9. If it is the last threat the state should not longer be in the fixing state


### PR submission checklist:
